### PR TITLE
Overhaul README

### PR DESCRIPTION
**Pull request by jdidion**
* Put sentences on separate lines to simplify future PR reviews
* Clean up the resources sections, and add things that were previously buried in READMEs in subfolders
* Add some missing implementations, tools, etc
* Removed mention of Atom plugin since Atom is deprecated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,105 +1,149 @@
...
 
 ### IDE Support
 
-- Visual Studio Code: [WDL Syntax Highlighter](https://marketplace.visualstudio.com/items?itemName=broadinstitute.wdl)
-- JetBrains IDE's: [Winstanly](https://plugins.jetbrains.com/plugin/8154-winstanley-wdl)
-- Atom: [Language-WDL](https://atom.io/packages/language-wdl)
-- Vim: [vim-wdl](https://github.com/broadinstitute/vim-wdl)
+| IDE                | Tool                                                                                             |
+| ------------------ | ------------------------------------------------------------------------------------------------ |
+| Atom               | [Language-WDL](https://atom.io/packages/language-wdl)                                            |
```
Comment (vsmalladi) [inline README.md]: Atom is deprecated. So I would remove this. https://github.blog/2022-06-08-sunsetting-atom/

File: README.md [hunk 2]
```diff
@@ -1,105 +1,149 @@
...
+| Emacs              | [poly-wdl](https://github.com/jmonlong/poly-wdl)                                                 |
+| Emacs              | [wdl-mode](https://github.com/zhanxw/wdl-mode)                                                   |
+| JetBrains          | [Winstanly](https://plugins.jetbrains.com/plugin/8154-winstanley-wdl)                            |
+| Sublime            | [WDL Syntax Highlighter](https://github.com/broadinstitute/wdl-sublime-syntax-highlighter)       |
+| Vim                | [vim-wdl](https://github.com/broadinstitute/vim-wdl)                                             |
+| Visual Studio Code | [WDL Syntax Highlighter](https://marketplace.visualstudio.com/items?itemName=broadinstitute.wdl) |
 
 ### Documentation
```
Comment (vsmalladi) [inline README.md:120]: Should you add mini-wdl and and the input_template: https://miniwdl.readthedocs.io/en/latest/input_template.html

File: README.md [hunk 3]
```diff
@@ -1,105 +1,149 @@
...
+- [Pytest-wdl](https://github.com/EliLillyCo/pytest-wdl) is a plugin for the pytest unit testing framework that enables testing of WDL tasks and workflows. Tests can be specified in either YAML format or python code.
 
 ### Packaging
 
-- [wdl-packager](https://github.com/biowdl/wdl-packager). WDL packaging utility that uses miniwdl to find which paths are imported and packages these into a zip 
-  together with the calling workflow. The zip can be used as an imports zip package for cromwell. The utility can add non-WDL files (such as the license) to the
-  zip package and provides options to package the zip in a binary reproducible way.
-
-# Contributing
+- [wdl-packager](https://github.com/biowdl/wdl-packager) packages a workflow and all of its imports into a zip file. The zip can be used as an imports zip package for cromwell. The utility can add non-WDL files (such as the license) to the zip package and provides options to package the zip in a binary reproducible way.
```
Comment (vsmalladi) [inline README.md:133]: I would add miniwdl zip: https://miniwdl.readthedocs.io/en/latest/zip.html

**Review by vsmalladi — CHANGES_REQUESTED** [review]
A few changes, but looks great

File: README.md (hunk 2 above)
Comment (jdidion) [reply README.md:120]: Not sure that belongs in IDE support - I mentioned it under language support instead

**Review by patmagee — APPROVED** [review]
LGTM, thank you for undertaking this work @jdidion

**Pull request merged**